wp-includes/widgets/class-wp-widget-media-image.php

```diff
@@ -317,10 +317,6 @@ public function render_control_template_scripts() {
 						echo sprintf( __( 'Current image: %s' ), '{{ data.currentFilename }}' );
 					?></p>
 				<# } #>
-			<# } else { #>
-				<div class="attachment-media-view">
-					<p class="placeholder"><?php echo esc_html( $this->l10n['no_media_selected'] ); ?></p>
```

Comment:
There is never an instance when we'd update the preview without an image, correct?